pkg/config/msp_test.go

```diff
@@ -8,16 +8,30 @@ package config
 
 import (
 	"bytes"
+	"crypto"
 	"crypto/ecdsa"
+	"crypto/elliptic"
 	"crypto/x509"
 	"crypto/x509/pkix"
+	"encoding/asn1"
+	"errors"
+	"log"
+	"math/big"
 	"testing"
 
+	"github.com/golang/protobuf/proto"
+	cb "github.com/hyperledger/fabric-protos-go/common"
 	mb "github.com/hyperledger/fabric-protos-go/msp"
 	"github.com/hyperledger/fabric/common/tools/protolator"
+	"github.com/hyperledger/fabric/pkg/config/mock"
 	. "github.com/onsi/gomega"
 )
 
+//go:generate counterfeiter -o mock/signer.go --fake-name Signer . signer
+type signer interface {
```

Comment:
Why do we need a fake signer? Seems it’s quite easy to create a real signer and the only ref I see below uses it in the happy path.

Reply:
Done